api/Navigator.json

```diff
@@ -364,10 +364,10 @@
               "version_added": false
             },
             "safari": {
-              "version_added": false
+              "version_added": "14"
```

Comment:
Tried calling `navigator.canShare()` in Safari 14.1, it returns false, so I can confirm it's shipped. Interesting!